contract_compassion/tests/test_base_module.py

```diff
@@ -20,8 +20,8 @@ class test_base_module(test_base_contract):
 
     def setUp(self):
         # Retrieve of income account
-        account_obj = self.env['account.account']
-        property_account_income = account_obj.search([
+        super(test_base_module, self).setUp()

```

Comment:
if i don't add it, i have an error saying that test_sponsorship_compassion or test_crm_compassion object have no env attribute when i do a simple self.env[something]